docs/services/services-guide-aws.md

```diff
@@ -0,0 +1,827 @@
+```
+
+After creating the security group, we add a rule to allow traffic at `MySQL` port
+
+```bash
+aws ec2 authorize-security-group-ingress \
+  --group-id="${RDS_SECURITY_GROUP_ID}" \
+  --protocol=tcp \
+  --port=3306 \
+  --region="${REGION}" \
+  --cidr=0.0.0.0/0  > /dev/null
+```
+
+1. A **DB Subnet Goup** needs to be created, so that the RDS instance is associated with different availability zones
+
+```bash
+
+DB_SUBNET_GROUP_NAME=crossplane-example-dbsubnetgroup
+
+# convert subnets to a white space separated list
+SUBNETS_LIST="${SUBNET_IDS//,/ }"
+
+aws rds create-db-subnet-group \
+  --region="${REGION}" \
+  --db-subnet-group-name="${DB_SUBNET_GROUP_NAME}" \
+  --db-subnet-group-description="crossplane-example db subnet group" \
+  --subnet-ids $SUBNETS_LIST > /dev/null
+
+```
+
+These resources later will be used to create cloud-specitic MySQL resources.
+
+### Set Up Crossplane
+
+Using the newly provisioned cluster:
+
+1. Install Crossplane from alpha channel. (See the [Crossplane Installation
+   Guide][crossplane-install] for more information.)
+```bash
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+2. Install the AWS stack into Crossplane. (See the [AWS stack
+   section][aws-stack-install] of the install guide for more information.)
+
+```yaml
+cat > stack-aws.yaml <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aws
+---
+apiVersion: stacks.crossplane.io/v1alpha1
+kind: StackRequest
+metadata:
+  name: stack-aws
+  namespace: crossplane-system
+spec:
+  package: "crossplane/stack-aws:master"
+EOF
+
+kubectl apply -f stack-aws.yaml
+```
+
+3. Obtain AWS credentials. (See the [Cloud Provider Credentials][cloud-creds]
+   docs for more information.)
+
+#### Infrastructure Namespaces
+
+Kubernetes namespaces allow for separation of environments within your cluster.
+You may choose to use namespaces to group resources by team, application, or any
+other logical distinction. For this guide, we will create a namespace called
+`app-project1-dev`, which we will use to group our AWS infrastructure
+components.
+
+* Define a `Namespace` in `aws-infra-dev-namespace.yaml` and create it:
+```yaml
+cat > aws-infra-dev-namespace.yaml <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aws-infra-dev
+EOF
+
+kubectl apply -f aws-infra-dev-namespace.yam
+```
+
+* You should see the following output:
+
+> namespace/aws-infra-dev.yaml created
+
+#### AWS Provider
+
+It is essential to make sure that the AWS user credentials is configured
+in Crossplane as a provider. Please follow the steps [provider guide][aws-provider-guide]
+for more information.
+
+#### Cloud-Specific Resource Classes
+
+Cloud-specific resource classes are used to define a reusable configuration for
+a specific managed service. Wordpress requires a MySQL database, which can be
```

Comment:
```suggestion
a specific managed resource. Wordpress requires a MySQL database, which can be
```

I think we more commonly say managed resource (meaning the actual `RDSInstance`), but managed service makes sense here too if you mean for our RDS support in general.